.travis.yml

```diff
@@ -17,5 +17,6 @@ install:
   - cd packages/forgae-cli
   - yarn link
   - cd ../..
+  - tsc
```

Comment:
I do not believe tsc is neccessary, as we already should have the converted js files in the dist folder